src/StoreApi/docs/products.md

```diff
@@ -63,8 +63,8 @@ GET /products?return_rating_counts=true
 | `tag`                                       | string  |    no    | Limit result set to products assigned a specific tag ID.                                                                                                                   |
 | `tag_operator`                              | string  |    no    | Operator to compare product tags. Allowed values: `in`, `not_in`, `and`                                                                                                    |
 | `on_sale`                                   | boolean |    no    | Limit result set to products on sale.                                                                                                                                      |
-| `min_price`                                 | string  |    no    | Limit result set to products based on a minimum price, provided using the smallest unit of the currency.                                                                   |
-| `max_price`                                 | string  |    no    | Limit result set to products based on a maximum price, provided using the smallest unit of the currency.                                                                   |
+| `min_price`                                 | string  |    no    | Limit result set to products based on a minimum price, provided using the smallest unit of the currency. The price needs to include 2 decimals (e.g. 100 will be 10000).   |
+| `max_price`                                 | string  |    no    | Limit result set to products based on a maximum price, provided using the smallest unit of the currency. The price needs to include 2 decimals (e.g. 100 will be 10000).   |
```

Comment:
I see your point in the improvement of docs, but we need to keep in mind, not every currency has two decimals, hence this text:

> "...provided using the smallest unit of the currency."

I understand it may be confusing, but given the above, the example should be more generic as well or I'd suggest showing the example of specific currencies, e.g:
- two-decimals currency like USD
- zero-decimal currency like JPY.

What do you think?

Reply:
I agree with you 100%. I have made the doc more generic. Tell me what you think